src/Modules/OAuth/README.md

```diff
@@ -0,0 +1,62 @@
+# OAuth Module
+The oAuth module provides a way to authenticate users using Microsoft Azure.
+
+## Prerequisites
+
+### Azure Application
+To use this module you need to create an application in Azure. You can do this at the [Azure Portal](https://portal.azure.com).
```

Comment:
> You will need to create an App Registration in Azure Active Directory. You can follows the steps at the official documentation: [Create a Microsoft Entra application](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
> 
> You will need to name your application. Use something descriptive, for instance the address of your website. Or "Website CMS login"
> 
> A Redirect URI needs to be set. The type is web and the value is `https://<your-domain>/private/connect/azure/check`. So it will look something like `https://www.the-adress-of-your-website.be/private/connect/azure/check`.
> 
> To be able to configure the application in Fork CMS you need the following data: 
> * `Application (client) ID`
> * `Application secret`
> * `Directory (tenant) ID`
>